### ISSUE TITLE
 npx create-nx-workspace@latest --preset=@analogjs/platform fails to install dependencies

### ISSUE THREAD

**Issue by DonaldMurillo** [analogjs/analog#396]
### Please provide the environment you discovered this bug in.

Windows 10 Pro Version 22H2
Processor 11th Gen Intel(R) Core(TM) i9-11900K @ 3.50GHz   3.50 GHz
Terminal Powershell

Angular CLI: 15.2.6
Node: 16.19.0
Package Manager: npm 8.19.3
OS: win32 x64

Angular:
...

Package                      Version
------------------------------------------------------
@angular-devkit/architect    0.1502.6 (cli-only)
@angular-devkit/core         15.2.6 (cli-only)
@angular-devkit/schematics   15.2.6 (cli-only)
@schematics/angular          15.2.6 (cli-only)

### Which area/package is the issue in?

create-analog

### Description

When running the preset command for a nx workspace, the installation fails as it is installing the angular packages.

### Please provide the exception or error you saw

```true
PS D:\Programming\tests> npx create-nx-workspace@latest --preset=@analogjs/platform --verbose

 >  NX   Let's create a new workspace [https://nx.dev/getting-started/intro]


✔ Installing dependencies with npm
✔ Successfully created the workspace: my-repo.

>  NX  Generating @analogjs/platform:preset

√ Add TailwindCSS for styling? (Y/n) · false
√ Add tRPC for typesafe client/server interaction? (y/N) · false
Angular has not been installed yet. Creating an Angular application
Fetching @nx/angular...
npm WARN deprecated stable@0.1.8: Modern JS already guarantees Array#sort() is a stable sort, so this library is deprecated. See the compatibility table on MDN: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/sort#browser_compatibility
npm WARN deprecated source-map-resolve@0.6.0: See https://github.com/lydell/source-map-resolve#deprecated
npm WARN deprecated sourcemap-codec@1.4.8: Please use @jridgewell/sourcemap-codec instead

added 895 packages, and audited 896 packages in 37s

95 packages are looking for funding
  run `npm fund` for details

found 0 vulnerabilities
Fetching @angular-devkit/core...

added 12 packages, and audited 13 packages in 14s

1 package is looking for funding
  run `npm fund` for details

found 0 vulnerabilities

 >  NX   generateFiles: No files found in "/Users/Dom/AppData/Local/Temp/tmp-7900-7Ah1ohRhYMYA/node_modules/@nx/js/src/generators/init/files". Are you sure you specified the correct path?


Error: generateFiles: No files found in "/Users/Dom/AppData/Local/Temp/tmp-7900-7Ah1ohRhYMYA/node_modules/@nx/js/src/generators/init/files". Are you sure you specified the correct path?
    at generateFiles (C:\Users\Dom\AppData\Local\Temp\tmp-7900-7Ah1ohRhYMYA\node_modules\@nx\devkit\src\generators\generate-files.js:35:15)
    at C:\Users\Dom\AppData\Local\Temp\tmp-7900-7Ah1ohRhYMYA\node_modules\@nx\js\src\generators\init\init.js:15:40
    at Generator.next (<anonymous>)
    at C:\Users\Dom\AppData\Local\Temp\tmp-7900-7Ah1ohRhYMYA\node_modules\tslib\tslib.js:167:75
    at new Promise (<anonymous>)
    at Object.__awaiter (C:\Users\Dom\AppData\Local\Temp\tmp-7900-7Ah1ohRhYMYA\node_modules\tslib\tslib.js:163:16)
    at initGenerator (C:\Users\Dom\AppData\Local\Temp\tmp-7900-7Ah1ohRhYMYA\node_modules\@nx\js\src\generators\init\init.js:11:20)
    at Object.<anonymous> (C:\Users\Dom\AppData\Local\Temp\tmp-7900-7Ah1ohRhYMYA\node_modules\@nx\angular\src\generators\init\init.js:36:53)
    at Generator.next (<anonymous>)
    at C:\Users\Dom\AppData\Local\Temp\tmp-7900-7Ah1ohRhYMYA\node_modules\tslib\tslib.js:167:75

 >  NX   Failed to apply preset: @analogjs/platform

   See above
```

### Other information

Workaround:
Create a new nx workspace integrated monorepo with angular
install the `@analogjs/platform` dep
run the new analog application generator

### I would be willing to submit a PR to fix this issue

- [ ] Yes
- [ ] No

**Comment by goetzrobin** [comment]
Thank you! It seems like this is an import path issue where something is not converted to a Windows friendly path...
Does `npx create-nx-workspace@latest` without the preset work without any issues on your machine?

**Comment by DonaldMurillo** [comment]
> 



> Thank you! It seems like this is an import path issue where something is not converted to a Windows friendly path...
> Does `npx create-nx-workspace@latest` without the preset work without any issues on your machine?

That command is the one I use every time I start a project on the. windows machine and does not give me any issues. I have used most of the integrated mono repo presets and they all work (from the cli menu)

**Comment by SOG-web** [comment]
am getting this error too

 >  NX   Cannot find module '../../utils/version-utils'

   Require stack:
   - /home/sogtheimmortal/Documents/Github/ROU/rou-education/node_modules/@analogjs/platform/src/lib/nx-plugin/src/generators/app/generator.js
   - /home/sogtheimmortal/Documents/Github/ROU/rou-education/node_modules/nx/src/config/workspaces.js
   - /home/sogtheimmortal/Documents/Github/ROU/rou-education/node_modules/nx/src/config/configuration.js
   - /home/sogtheimmortal/Documents/Github/ROU/rou-education/node_modules/nx/src/utils/package-manager.js
   - /home/sogtheimmortal/Documents/Github/ROU/rou-education/node_modules/nx/bin/init-local.js
   - /home/sogtheimmortal/Documents/Github/ROU/rou-education/node_modules/nx/bin/nx.js
   - /home/sogtheimmortal/.nvm/versions/node/v18.15.0/lib/node_modules/nx/bin/nx.js
   Pass --verbose to see the stacktrace.

**Comment by mainawycliffe** [comment]
Experiencing this as well, I was digging around and it seems that the utils directory, despite being included when nx-plugin is built in the output directory, it isn't uploaded to NPM, hence the error.

![image](https://github.com/analogjs/analog/assets/12270550/0517db7d-14de-4cda-b252-af12c4ae18ae)

I can try and work on a fix for this tomorrow if no one is. 

CC @brandonroberts

**Comment by mainawycliffe** [comment]
@SOG-web experiencing the same thing as well. Was digging around the code and it seems like the utils directory isn't uploaded to NPM despite being included in the final build of the nx-plugin.

![image](https://github.com/analogjs/analog/assets/12270550/3a5dfdb2-609d-439d-bffd-9b76a44af08c)

Let me see if I can track the reason why that's happening and with some luck, PR something

Any ideas @brandonroberts?

**Comment by goetzrobin** [comment]
Maybe this can help: https://codeburst.io/what-really-gets-packaged-f2b49397c2a5

**Comment by goetzrobin** [comment]
<img width="742" alt="Screenshot 2023-05-22 at 9 35 26 AM" src="https://github.com/analogjs/analog/assets/35136007/e4a74320-5d78-40fc-83ed-9601ba7ad216">
Very weird... running npm pack locally creates the files that are missing in npm. just leaving this for a reference... @brandonroberts

**Comment by brandonroberts** [comment]
Not sure what happened here, but I ran a new build and published as `0.2.0-beta.11` that includes the missing files. You can update using `ng update @analogjs/platform@latest` or `nx migrate @analogjs/platform@latest`

Hope this helps.
